docs/contributor/02-40-broker-configuration-for-given-plan.md

```diff
@@ -3,31 +3,24 @@
 > [!NOTE] 
 > This document has been deprecated. Kyma version configuration is not possible.
 
-Some processes in Kyma Environment Broker (KEB) can be configured to deliver different results. KEB needs a ConfigMap with a configuration for the given Kyma version and plan to process the requests. 
-A default configuration must be defined for the chosen Kyma version. This configuration must be recognized by KEB as applicable for all the supported plans. You can also set a separate configuration for each plan.
+Some processes in Kyma Environment Broker (KEB) can be configured to deliver different results. KEB needs a ConfigMap with a configuration for the given Kyma plan to process the requests. 
+A default configuration must be defined. This configuration must be recognized by KEB as applicable for all the supported plans. You can also set a separate configuration for each plan.
   
-While processing requests, KEB reads a configuration from a ConfigMap which holds data about processable Kyma versions and configurable units for a given plan. Only one ConfigMap can exist for a given Kyma version, but it also can be set for multiple Kyma versions if the configuration is the same for every targeted version.
-
+While processing requests, KEB reads a configuration from a ConfigMap which holds data for a given plan.
 > [!NOTE] 
 > Create all configurations in the `kcp-system` namespace.
 
 > [!NOTE] 
 > Currently, only the Kyma custom resource template and the additional components list can be configured.
```

Comment:
You have removed the `additional components` from the code, so is the info about them in this note still valid?